flink-formats/flink-csv/src/test/java/org/apache/flink/formats/csv/CsvRowDataSerDeSchemaTest.java

```diff
@@ -299,6 +299,27 @@ public void testSerializeDeserializeNestedTypes() throws Exception {
         testSerDeConsistency(nullRow, serSchemaBuilder, deserSchemaBuilder);
     }
 
+    @Test
+    @SuppressWarnings("unchecked")
+    public void testDeserializationWithDisableQuoteCharacter() throws Exception {
```

Comment:
We can simplify the test into following to reuse the logic of constructing schema and rows. 

```scala
    @Test
    public void testDeserializationWithQuoteCharacterDisabled() throws Exception {
        Consumer<CsvRowDataDeserializationSchema.Builder> deserConfig =
                (deserSchemaBuilder) ->
                        deserSchemaBuilder
                                .setEscapeCharacter('*')
                                .disableQuoteCharacter()
                                .setArrayElementDelimiter(":")
                                .setFieldDelimiter(';');

        testFieldDeserialization(STRING(), "\"abc", "\"abc", deserConfig, ";");
    }
```